web/cspell.json

```diff
@@ -75,6 +75,7 @@
         "textinput",
         "tkip",
         "udev",
+	      "unpartitioned",
```

Comment:
indentation looks strange on github

Reply:
It is strange, I see it correctly in the code.